### DISCUSSION TITLE
Different RLS to view vs table

### DISCUSSION THREAD

**Discussion by MariuzM** [supabase/supabase#16124] [Questions]
In profiles `table` i have that only `authenticated` with matching `auth_uid` of the row can see the row of that table. 
But in `view` i want any `authenticated` user to see all data as long as its `authenticated`.

I have tried with `security_invoker` but that just uses the same RLS as table. 
```
CREATE VIEW
  profiles_view
WITH
  (security_invoker) AS
SELECT
  id,
  first_name,
FROM
  public.profiles;
```

I have also tried changing owner but getting this error 
<img width="647" alt="image" src="https://github.com/supabase/supabase/assets/36937039/6af28031-bb8b-4ded-9d10-60f1aecfda17">

**Comment by ilbertt** [comment]
I need to achieve the same, but I also don't know how to do it
